app/presenters/setting_presenter.rb

```diff
@@ -52,10 +52,18 @@ def settings_type
     attribute(:settings_type) || Setting.setting_type_from_value(default)
   end
 
+  def matches_search_query?(query)
+    if (res = query.match(/name\s*=\s*(\S+)/))
+      name == res[1]
+    else
+      description.include?(query) || name.include?(query) || full_name&.include?(query)
+    end
+  end
```

Comment:
This will break compatibility, users do expect scoped_search. I have no problem with that, I think this is a minor problem. Just saying.